core/src/main/scala/pureconfig/ConfigCursor.scala

```diff
@@ -82,6 +82,21 @@ sealed trait ConfigCursor {
       case v => Left(CannotConvert(v.toString, "Long", "Unable to convert Number to Long"))
     })
 
+  /**
+   * Casts this cursor to a long representing a memory size according to the way com.typesafe.config.Config#getBytes(java.lang.String) does.
+   *
+   * @return a `Right` with the memory size value pointed to by this cursor if the cast can be done, `Left` with a list of
+   *         failures otherwise.
+   */
+  def asBytes: ConfigReader.Result[Long] = scopeFailure {
+    ConvertHelpers.tryToEither {
+      Try {
+        val wrapped = value.atKey("_")
+        wrapped.getBytes("_")
```

Comment:
This is a bit unfortunate, but it seems necessary...

Reply:
The alternative would be to copy their bytes parsing code and convert it to PureConfig standards (e.g. using either along the way instead of throwing). We did this with [Duration](https://github.com/pureconfig/pureconfig/blob/master/core/src/main/scala/pureconfig/DurationUtils.scala) and [Period](https://github.com/pureconfig/pureconfig/blob/master/core/src/main/scala/pureconfig/PeriodUtils.scala). I'm ok with using Typesafe Config's implementation for now though.